Add `fastlane-env-reminder` GitHub Action (#2)
fastlane/github-actions: fastlane-env-reminder/docs/contributors.md

# Contributors

### Build and Test

```bash
$ npm run build
$ npm test
```

> ### Checkin

Reviewer: ```suggestion
### Check in
```

> - Do checkin source (src)

Reviewer: ```suggestion
- Do checkin source (`src`)
```

> - Do checkin build output (lib)

Reviewer: ```suggestion
- Do checkin build output (`lib`)
```

- Do not checkin runtime node_modules in development workflow (see [Dev Workflow documentation](https://github.com/actions/toolkit/blob/master/docs/javascript-action.md#dev-workflow))

> - Do not checkin devDependency node_modules (husky can help see below)

Reviewer: ```suggestion
- Do not check in devDependency `node_modules` (husky can help, see below)
```

although this might not be necessary with the sentence before

> ### devDependencies

Reviewer: ```suggestion
### `devDependencies`
```

> In order to handle correctly checking in node_modules without devDependencies, we run [Husky](https://github.com/typicode/husky) before each commit.

Reviewer: ```suggestion
In order to handle correctly checking in `node_modules` without `devDependencies`, we run [Husky](https://github.com/typicode/husky) before each commit.
```
is this still true?

> This step ensures that formatting and checkin rules are followed and that devDependencies are excluded. To make sure Husky runs correctly, please use the following workflow:

Reviewer: ```suggestion
This step ensures that formatting and checkin rules are followed and that `devDependencies` are excluded. To make sure Husky runs correctly, please use the following workflow:
```

```
npm install                                 # installs all devDependencies including Husky
git add abc.ext                             # Add the files you've changed. This should include files in src, lib, and node_modules (see above)
git commit -m "Informative commit message"  # Commit. This will run Husky
```

> During the commit step, Husky will take care of formatting all files with [Prettier](https://github.com/prettier/prettier) as well as pruning out devDependencies using `npm prune --production`.

Reviewer: ```suggestion
During the commit step, Husky will take care of formatting all files with [Prettier](https://github.com/prettier/prettier) as well as pruning out `devDependencies` using `npm prune --production`.
```
